### DISCUSSION TITLE
Tab vs Segment

### DISCUSSION THREAD

**Discussion by stanlrt** [chakra-ui/ark#2626] [Q&A] [answered]
I am not sure when to use Segments over Tabs. What are the concrete implementation differences (what's is and isn't possible with each option) ?

**Comment by segunadebayo** [comment, marked as answer]
Segment Control: a fancy looking radio group element you can use within forms.

Tabs: used for sectioning or grouping content

**Reply by stanlrt** [reply]
Thanks!
